libraries/matrix/impl/src/main/kotlin/io/element/android/libraries/matrix/impl/timeline/item/event/EventMessageMapper.kt

```diff
@@ -69,7 +69,7 @@ class EventMessageMapper {
         }
         MessageContent(
             body = message.body(),
-            inReplyTo = message.inReplyTo()?.let { UserId(it) },
+            inReplyTo = message.inReplyTo()?.eventId?.let { EventId(it) },
```

Comment:
We have `fun String.asEventId()` that does more check. Maybe change to

```suggestion
            inReplyTo = message.inReplyTo()?.eventId?.let {it.asEventId() },
```

?

Reply:
I can change it, but this is what I currently see for `String.asEventId()`:

```kotlin
fun String.asEventId() = EventId(this)
```

Should it be doing anything else?

Reply:
The check is added on  an other PR https://github.com/vector-im/element-x-android/pull/315